main/zoe/guide/contracts/callSpread.md

```diff
@@ -0,0 +1,77 @@
+
+This option is settled financially. There is no requirement that the original purchaser have
+ownership of the underlying asset at the start, and the beneficiaries shouldn't expect to take
+delivery at closing.
+
+It is common for the strike price and collateral to use the same currency, though this contract
+decouples them. This would be like a spread based on the price of APPL stock (`Underlying`),
+where the stock price is measured in USD (`Strike`), but the contract pays out in JPY
+(`Collateral`).
+
+The issuerKeywordRecord specifies the issuers for four keywords: Underlying, Strike, Collateral,
+and Quote. The original deposit and the payout are in Collateral. Strike amounts are used for the
+price oracle's quote as to the value of the Underlying, as well as the strike prices in the
+terms. The priceAuthority's quotes are ERTP assets issued by a QuoteAuthority, which must be
+specified using the keyword Quote.
+
+The terms include { timer, underlyingAmount, expiration, priceAuthority, strikePrice1,
+strikePrice2, settlementAmount }. `timer` must be recognized by `priceAuthority`. `expiration` is a
+time recognized by the `timer`. `underlyingAmount` is passed to `priceAuthority`, so it could be an
+NFT or a fungible amount. `strikePrice2` must be greater than `strikePrice1`. `settlementAmount`
+uses Collateral.
+
+<<< @/snippets/zoe/contracts/test-callSpread.js#startInstance
+
+The terms specify all the details of the options that will be created, but the options are not
+handed out until the creator funds the collateral that will make them valuable.  The
+creatorInvitation has customProperties that include the amounts of the two options as longAmount
+and shortAmount.
+
+<<< @/snippets/zoe/contracts/test-callSpread.js#invitationDetails
+
+The funder can use these option amounts to create an offer that assures them they will get the two
+options in exchange for the funds. The proposal describes the options they want and the collateral
+they give. When the offer is made they get a payout that contains the two option positions, which
+are themselves invitations which can be exercised for free, and provide the option payouts with the
+keyword Collateral.
+
+<<< @/snippets/zoe/contracts/test-callSpread.js#creatorInvitation
+
+The options that are returned by the contract are Zoe invitations, so their values and terms can be
+verified by asking Zoe for the contract terms.  This makes it possible to sell the options because
+a prospetive purchaser can inspect their value. The prospective purchaser can verify the underlying
+amount, and check to see that the priceAuthority is one they are willing to rely on. They can check
+the expiration to see that it matches their expectations (here it's a small integer suitable for a
+manual timer in a test; in actual use, it would represent block height or wall clock time.) The
+strike prices and settlement amount are likewise visible.
```

Comment:
```suggestion
The options returned by the contract are Zoe invitations, so their values and terms can be
verified by asking Zoe for the contract terms.  This means you can sell the options because
a prospective purchaser can inspect their value by verifying the underlying
amount, and checking that they are willing to rely on this priceAuthority. They can check
that the expiration matches their expectations (here it's a small integer suitable for a
manual timer in a test; in actual use, it represents block height or wall clock time.) The
strike prices and settlement amount are likewise visible.
```

Reply:
reworded considerably.